reflex/components/radix/themes/components/text_area.py

```diff
@@ -83,19 +83,19 @@ class TextArea(RadixThemesComponent, elements.Textarea):
     wrap: Var[str]
 
     # Fired when the value of the textarea changes.
-    on_change: EventHandler[lambda e0: [e0.target.value]]
+    on_change: EventHandler[input_event]
```

Comment:
if this class inherits from `elements.Textarea`, do we need these definitions here? are they for autodoc purposes?

Reply:
Soon TextField will inherit from `elements.Input`, so similar question there.

Reply:
removed unnecessary ones! should work fine as PYI file remained the same